### PR TITLE
Add error_code

### PR DESCRIPTION
**Pull request by Megamouse**
Adds error_code to cellGem and fixes a log message.
Adds error_code to cellMouse.
Adds error_code to cellVdec.

### CODE REVIEW

File: rpcs3/Emu/Cell/Modules/cellGem.cpp [hunk 1]
```diff
@@ -843,7 +888,7 @@ s32 cellGemIsTrackableHue(u32 hue)
 	return true;
```
Comment (RipleyTom) [inline rpcs3/Emu/Cell/Modules/cellGem.cpp]: Probably should be not_an_error(true);

File: rpcs3/Emu/Cell/Modules/cellGem.cpp [hunk 2]
```diff
@@ -636,7 +681,7 @@ s32 cellGemGetMemorySize(s32 max_connect)
 	return max_connect <= 2 ? 0x120000 : 0x140000;
```
Comment (RipleyTom) [inline rpcs3/Emu/Cell/Modules/cellGem.cpp]: same here and all the other that don't return CELL_OK

Comment (Megamouse) [reply rpcs3/Emu/Cell/Modules/cellGem.cpp]: I totally missed that one. thx

File: rpcs3/Emu/Cell/Modules/cellGem.cpp (hunk 1 above)
Comment (Megamouse) [reply rpcs3/Emu/Cell/Modules/cellGem.cpp]: the function is still s32

**Comment by incognitoh** [comment]
hi @Megamouse @Nekotekina 

seems like Daytona USA NPUB30493 regressed with this pr.  playable > nothing.  the game now closes immediately after booting. cleared all caches, tried several different settings, same result...

**Comment by incognitoh** [comment]
hi @Megamouse  @Nekotekina 

seems like Daytona USA NPUB30493 regressed with this pr after it was merged into master.  the game now closes immediately after booting.  I cleared all caches, tried several different settings, same result...hopefully others can test it to confirm. This is on Windows 10.

RPCS3 v0.0.7-8667-cb00b71c Alpha > closes immediately at boot/startup
previous Master builds > playable

**Comment by incognitoh** [comment]
same thing is happening to Fighting Vipers NPJB00252...

**Comment by incognitoh** [comment]
also happening with Sonic the Fighters and Virtua Fighter 2. the pattern here is that this PR regressed all these titles which are all SEGA games.

**Comment by Megamouse** [comment]
Can you please stop spamming this PR?
There is no relation between the regressions and this code.

**Pull request merged**